ValveAnnulusAnalysis/HeartValveLib/util.py

```diff
@@ -3,6 +3,37 @@
 #
 
 
+class Signal:
```

Comment:
It is very nice and simple and an immediate solution to that messy problem with the Qt signals. Could you please add a few sentences as class documentation that explains why this class is needed and and a reference to the Slicer forum discussions?